etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,369 @@
+---
+#########################################
+# Stackhpc default firewall configuration
+#########################################
+
+stackhpc_firewalld_zones: |
+  {% set network_zones = [] %}
+  {% for network in network_interfaces %}
+  {% if network | net_zone is not none %}
```

Comment:
I think this would work and would catch an empty string:
```
{% if network | net_zone %}
```

Reply:
Neato